tests/integration/targets/nxos_static_routes/defaults/main.yaml

```diff
@@ -0,0 +1,2 @@
+---
+testcase: "*"
```

Comment:
```suggestion
testcase: '[^_].*'
```

Reply:
Also, please add an underscore before remove_config and populate_config, so we don't run them as test files.

Reply:
This needs to be updated.